.travis.yml

```diff
@@ -9,9 +9,7 @@ os:
 env:
   - "PATH=/home/travis/gopath/bin:$PATH"
 script:
-  - test -z "$(gofmt -l ./)"
-  - test -z "$(go vet -v ./...)"
-  - go build
```

Comment:
Yeah, unless `go test` already does it, the `gofmt` check should definitely remain there. It's already caught a couple misformatted PRs.